registration/tests/serializers/test_family_detail.py

```diff
@@ -67,16 +61,3 @@ def test_family_detail_serializer_guests(self):
         self.assertEqual(
             data["guests"], [StudentSerializer(self.guest, context=context).data]
         )
-
-    def test_family_detail_serializer_interactions(self):
```

Comment:
are we planning to rewrite a test later?

Reply:
not really 😅 there was only a test before because there was custom logic, but since we removed all that I don't think we need to keep it around